src/components/Link/link.module.scss

```diff
@@ -78,6 +77,7 @@
   &.primary,
   &.secondary,
   &.disruptive {
+    display: inline-block;
```

Comment:
seems undesirable to have these variants provide a different display property compared to a standard link.

Reply:
FYI, removing this will break `focus-visible` styles for `Link`. It's not a regression as current versions `focus-visible` styles are busted unless `Link` is given either a `display` type or `width` during implementation, either as inline styles or a CSS override, which should not be necessary IMHO.

Reply:
i dont' want the app having to manage it. We will proceed with this since the default (non-variant) version shouldn't really be used. perhaps we should make primary the default as a breaking change in the near future?